iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/methods/DeviceMethodTests.java

```diff
@@ -340,7 +339,7 @@ public void invokeMethodDefaultConnectionTimeoutSucceed() throws Exception
         Assert.assertEquals(buildExceptionMessage("Unexpected status errors occurred", testInstance.deviceTestManager.client), 0, deviceTestManger.getStatusError());
     }
 
-    @Test (expected = IotHubGatewayTimeoutException.class)
```

Comment:
With the throttle resistant test rule, we can no longer use this method of checking for an exception upon executing some code. If the test encounters a 429 exception, that 429 gets compared to this GatewayTimeoutException, and since they don't match, the test throws an AssertionFailedException. Once that AssertionFailedException bubbles up to the throttle resistant test rule, the test won't be re-run since the test didn't seem to fail from a 429.